roles/ceph-monitor/files/92-ceph

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Please move this file to the proper directory in Ursula.  File and templates should be pathed for their final destination:
roles/ceph-monitor/files/etc/update-motd.d/92-ceph